package.json

```diff
@@ -14,7 +14,7 @@
     "bundle:react": "vite build && npm run makemessages && npm run compilemessages",
     "check-linting": "prettier src/open_inwoner/js src/open_inwoner/scss src/open_inwoner/react src/open_inwoner/webcomponents -c",
     "collect": "cp -r node_modules/leaflet/dist/images src/open_inwoner/static/bundles && cp -r node_modules/material-icons/iconfont/*.woff src/open_inwoner/static/bundles && cp -r node_modules/material-icons/iconfont/*.woff2 src/open_inwoner/static/bundles && cp -r node_modules/@fortawesome/fontawesome-free/webfonts src/open_inwoner/static/",
-    "compilemessages": "formatjs compile-folder --ast src/open_inwoner/react/i18n/locales src/open_inwoner/react/i18n/compiled",
+    "compilemessages": "formatjs compile-folder --ast src/open_inwoner/react/i18n/locales src/open_inwoner/react/i18n/compiled && pre-commit run end-of-file-fixer --files src/open_inwoner/react/i18n/compiled/*.json || true",
```

Comment:
Actually we do not need this PR at all,
you should just run `pre-commit` locally and you' ll see it's solved.

We do have this new large formatting PR here in order to put all of the formatting together: https://github.com/maykinmedia/open-inwoner/pull/2011